20.ValidParentheses_251019.md

```
@@ -0,0 +1,181 @@
+        for char in s:
+            if char in open_to_close:
+                open_brackets_stack.append(char)
+                correct_s = correct_s + char
+            else:
+                correct_s = correct_s + (open_to_close[open_brackets_stack.pop()] if open_brackets_stack else "#" )
+        return s == correct_s and len(open_brackets_stack) == 0
+```
+
+
+他の方の回答を見る
+- https://github.com/h1rosaka/arai60/pull/8/files
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)